x-pack/legacy/plugins/monitoring/public/directives/main/index.html

```diff
@@ -1,5 +1,5 @@
+<div id="internal-monitoring-warning" style="margin: 16px 8px 16px;"></div>
```

Comment:
What pages are you trying to get this warning to show up on? This location seems like it's global but setup mode isn't available on every page

Reply:
It should still show up the same as the setup button did. Are you seeing a different behavior?

Reply:
Yea, that's not all the pages though. We only show the setup mode button on the pages where setup mode does something. Did you mean for it to show up on only those pages, or on all pages?

Reply:
It should **not** show up on all the pages. The only difference here is that we don't inherit from `app-container` any more. This should still show (and hide) on the same pages the setup button did. Are you seeing a different behavior?

Reply:
I'm not actually. I'm seeing it only show up on specific pages, which seems strange. I'm actually not sure the right answer here, but a deprecation warning should probably show up on every page? Maybe?